src/applications/burials/tests/config/documents.unit.spec.jsx

```diff
@@ -18,7 +18,7 @@ describe('Burials document upload', () => {
   } = formConfig.chapters.additionalInformation.pages.documentUpload;
 
   it('should render', () => {
-    const form = ReactTestUtils.renderIntoDocument(
+    const form = render(
```

Comment:
Using `renderIntoDocument` with `Provider` seems to result in a null return value, using RTL works with minimal changes.